deepmedic/frontEnd/configParsing/trainSessionParams.py

```diff
@@ -324,6 +324,19 @@ def __init__(self,
         # Preprocessing
         self.pad_input_imgs = cfg[cfg.PAD_INPUT] if cfg[cfg.PAD_INPUT] is not None else True
 
+        # Normalisation
+        self.norm = cfg[cfg.NORM]
```

Comment:
We should add a comment saying what values are expected? At least if it's not in the config file itself.